htdocs/web_portal/views/site/edit_api_auth.php

```diff
@@ -67,6 +94,45 @@
                 <label class="input_label" for="ALLOW_WRITE">Allow API write</label>
             </div>
         </div>
-        <input type="submit" value="Edit credential" class="input_button">
+        <?php } else { ?>
+        <div>
+            <p>
+                Note: If you wish to edit the content for either Identifier
+                or Credential type. Please, visit the
+                <?php
+                echo('<a href="');
+                xecho(\GocContextPath::getPath());
+                echo('index.php?Page_Type=Edit_API_Authentication_Entity');
+                echo('&authentityid=');
+                echo($params['authEnt']->getId());
```

Comment:
```suggestion
                xecho($params['authEnt']->getId());
```

It's probably not a bad thing to use `xecho` here, and it would be consitent with elsewhere in this file.